python/src/GaussLegendre_doc.i.in

```diff
@@ -9,27 +9,87 @@ Available constructors:
+    
+    The default discretization is *GaussLegendre-DefaultMarginalIntegrationPointsNumber* in each dimension, see :class:`~openturns.ResourceMap`.
 
 Notes
 -----
-The Gauss-Legendre algorithm enables to approximate the definite integral:
+We consider a function :math:`f: \Rset^d \mapsto \Rset^p` and a domain :math:`\set{D} = [\vect{a}, \vect{b}]`. The Gauss-Legendre algorithm enables to approximate the definite integral:
 
 .. math::
+    :label: intInit
 
-    \int_{\vect{a}}^\vect{b} f(\vect{t})\di{\vect{t}}
+    \int_{\vect{a}}^{\vect{b}} f(\vect{t})\di{\vect{t}}
 
 
-with :math:`f: \Rset^d \mapsto \Rset^p`, :math:`\vect{a}, \vect{b}\in\Rset^d` using a fixed tensorized Gauss-Legendre approximation:
+using a fixed tensorized Gauss-Legendre approximation. The approximation is:
 
 .. math::
+    :label: approxInt
 
-    \int_{\vect{a}}^\vect{b} f(\vect{t})\di{\vect{t}} = \sum_{\vect{n}\in \cN}\left(\prod_{i=1}^d w^{N_i}_{n_i}\right)f(\xi^{N_1}_{n_1},\dots,\xi^{N_d}_{n_d})
+    \int_{\vect{a}}^{\vect{b}} f(\vect{t}) \di{\vect{t}} = \sum_{n_1=1}^{N_1} \dots  \sum_{n_d=1}^{N_d}\left( \prod_{i=1}^d w^{N_i}_{n_i}\right) f(\xi^{N_1}_{n_1},\dots,\xi^{N_d}_{n_d})
 
-where :math:`\xi^{N_i}_{n_i}` is the :math:`n_i` th node of the :math:`N_i` -points Gauss-Legendre 1D integration rule and :math:`w^{N_i}_{n_i}` the associated weight.
+where :math:`\xi^{N_i}_{n_i}` is the :math:`n_i` -th node computed from the :math:`N_i` -points Gauss-Legendre 1D integration rule and :math:`w^{N_i}_{n_i}` the associated weight.
 
+To use that approximation, the integration variables are scaled to the domain :math:`[0,1]^d` with the mapping function denoted :math:`\varphi` and defined by:
+
+.. math::
+    :label: mapping
+
+    \varphi : & [0,1]^d  \rightarrow [\vect{a}, \vect{b}]\\
+           & \vect{x} \rightarrow \vect{t} = \vect{a} + (\vect{b}-\vect{a}) \vect{x}
+
+where each component is :math:`t_i = a_i + (b_i-a_i)x_i` for :math:`1 \leq i \leq d`.
+
+We denote by :math:`|\vect{b}-\vect{a}| = \prod_{i=1}^d (b_i-a_i)` the volume of the domain :math:`[\vect{a}, \vect{b}]`. Then, we have:
+
+.. math::
+    :label: equivInt
+
+    \int_{\vect{a}}^{\vect{b}} f(\vect{t})\di{\vect{t}} & = \int_{[0,1]^d} |\vect{b}-\vect{a}| f\left( \vect{a} + (\vect{b}-\vect{a})\vect{x}\right)\di{\vect{x}} \\
+    & = \int_{[0,1]^d} \tilde{f}(\vect{x})\di{\vect{x}}
+
+where we introduced the scaled function :math:`\tilde{f}` on :math:`[0,1]^d` defined by:
+
+.. math::
+    :label: scaledFunc
+
+    \tilde{f}(\vect{x}) = |\vect{b}-\vect{a}| f \circ \varphi ( \vect{x} )
+
+Then, the Gauss-Legendre quadrature rule is applied to the function :math:`\tilde{f}` defined on :math:`[0,1]^d`. We denote by :math:`\tilde{\xi}^{N_i}_{n_i}` the :math:`n_i` -th node of the :math:`N_i` -points Gauss-Legendre 1D integration rule and :math:`\tilde{w}^{N_i}_{n_i}` the associated weight. Then we have:
```

Comment:
try to skip a line at end of sentences to shorten lines (easier to review, and diffs will be more readable in the future)

Reply:
also: rebase (never pull or merge)

Reply:
ok